color.go

```diff
@@ -18,8 +18,7 @@ var (
 	// or not. This is a global option and affects all colors. For more control
 	// over each color block use the methods DisableColor() individually.
 	NoColor = os.Getenv("TERM") == "dumb" ||
-		!isatty.IsTerminal(os.Stdout.Fd()) ||
-		!isatty.IsCygwinTerminal(os.Stdout.Fd())
+		(!isatty.IsTerminal(os.Stdout.Fd()) || !isatty.IsCygwinTerminal(os.Stdout.Fd()))
```

Comment:
Previously it was like:

```go
	NoColor = os.Getenv("TERM") == "dumb" ||
		!isatty.IsTerminal(os.Stdout.Fd()) ||
		!isatty.IsCygwinTerminal(os.Stdout.Fd())
```

Do you say it should be like this?

```go
	NoColor = os.Getenv("TERM") == "dumb" ||
		!isatty.IsTerminal(os.Stdout.Fd()) ||
		isatty.IsCygwinTerminal(os.Stdout.Fd())
```

I'm really confused right now. I don't have a way to test it, so can you test and give me the correct answer maybe?